pykechain/models/widgets/helpers.py

```diff
@@ -107,6 +108,200 @@ def _set_title(
     return meta, title
 
 
+def _set_description(
+        meta: Dict,
+        description: Optional[Union[Text, bool]] = None,
+        **kwargs
+) -> Tuple[Dict]:
+    """
+    Set the customDescription in the meta based on provided optional custom description or no description.
+
+    This will inject into the meta the `customDescription` and `showDescriptionValue` if the `description` is provided
+    as argument, otherwise it will inject 'no description'. It returns the meta definition of the widget.
+
+    :param meta: meta dictionary to augment
+    :type meta: dict
+    :param description: A description for the widget
+            * String value: use the description
+            * None or False: No description at all.
+    :type description: basestring or None or bool
+
+    :return: tuple of meta
+    :rtype: Tuple[Dict,Text]
```

Comment:
This rtype does not correspond with the typing hint in the function. There it is a `Tuple[Dict, Text]` (meaning a tuple containing a Dict and a String.

Reply:
Fixed